### PR TITLE
Make a test for custom cudnn environment

### PR DESCRIPTION
**Pull request by unnonouno**
I made a test for installing Chainer with cudnn which is installed a custom directory.

### CODE REVIEW

File: test_install.sh [hunk 1]
```diff
@@ -3,4 +3,8 @@
 timeout 20m pip install -vvvv chainer/dist/*.tar.gz --user
 
 # check if cupy is installed
-python -c 'import cupy'
+if [ $CUDNN = none ]; then
```
Comment (okuta) [inline test_install.sh]: Is `$CUDNN` declared?

How about adding `env['CUDNN'] = conf['cudnn']` to https://github.com/pfnet/chainer-test/pull/178/files#diff-97414e63442fe1e023eeb0b68b38eac5R67

Comment (unnonouno) [inline test_install.sh]: Do you mean that we need to make two types of install test scripts?

Comment (gwtnb) [inline test_install.sh]: How about making `$CUDNN` empty when you want to disable cuDNN and using following code?

```sh
if [ "$CUDNN" = none ]; then
```

Comment (gwtnb) [inline test_install.sh]: fixed

```sh
if [ -z "$CUDNN" ]; then
```

Comment (unnonouno) [inline test_install.sh]: ok

**Comment by okuta** [comment]
LGTM!

**Pull request merged**